processors/base/helpers/__init__.py

```diff
@@ -132,7 +136,7 @@ def find_list_of_identifiers(text):
 
     # Pattern could be improved based on a extended
     # clinical trial identifiers format analysis
-    PATTERN = r'(%s[\s]*[\d-]{3,})'
+    PATTERN = r'(%s\s*[\w\d-]{3,}(?:\s*\([\w]*\))?)'
```

Comment:
This has been changed because the previous one didn't match identifiers like `TAKEDA-AD-4833_409 (PHARMO)` at all.